src/display/lv_display_private.h

```diff
@@ -18,6 +18,10 @@ extern "C" {
 #include "../draw/lv_draw.h"
 #include "lv_display.h"
 
+#if LV_USE_PERF_MONITOR
```

Comment:
I believe this should read 


```c
#if LV_USE_SYSMON
```


The reason why I say that is because of either the perf monitor or the mem monitor is enabled then the file sill be included. This is because later on in this file both the memory monitor and also the perf monitor are check to see if they are enabled.